### PR TITLE
Added Section "Plugins and High Availability"

### PR DESCRIPTION
**Pull request by aaronrothschild**
- Noted auto install across servers
- Noted slight behavior change in 5.15

### CODE REVIEW

File: source/deployment/cluster.rst [hunk 1]
```diff
@@ -293,6 +293,14 @@ Mattermost runs periodic tasks via the `job server <https://docs.mattermost.com/
 
 :ref:`Run all job servers <command-line-tools-mattermost-jobserver>` with ``--noschedule flag``, then set ``JobSettings.RunScheduler`` to ``true`` in config.json for all app servers in the cluster. The cluster leader will then be responsible for scheduling recurring jobs.
 
+Plugins and High Availability
+^^^^^^^^^^^^^^^^
+
+As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. 
+
+Note a slight behavior change in v5.15:
```
Comment (amyblais) [inline source/deployment/cluster.rst]:
```suggestion
Note a slight behaviour change in 5.15:
```

File: source/deployment/cluster.rst [hunk 2]
```diff
@@ -293,6 +293,14 @@ Mattermost runs periodic tasks via the `job server <https://docs.mattermost.com/
...
+When you install a plugin in 5.14 -> enable -> uninstall -> install the same plugin again, the plugin's initial state remained  enabled. As of v5.15, when you install a plugin -> enable -> uninstall -> install the same plugin again, the plugin's initial state will now be DISABLED.
```
Comment (amyblais) [inline source/deployment/cluster.rst]:
```suggestion
When you install a plugin in 5.14 via **enable > uninstall > install the same plugin again**, the plugin's initial state remained  enabled. As of 5.15, when you install a plugin via **enable > uninstall > install the same plugin again**, the plugin's initial state will now be disabled.
```

Comment (amyblais) [reply source/deployment/cluster.rst]: This might need more details on where the "enable > uninstall > install the same plugin again" settings are.

Comment (ali-farooq0) [reply source/deployment/cluster.rst]: Does this seem awfully detailed? Should we rather say something like: `Plugins retained their previous state on reinstall. As of 5.15, plugins are always disabled on install.` Or something like that -- not a word wizard unfortunately :P

Comment (ali-farooq0) [reply source/deployment/cluster.rst]: Perhaps @justinegeffen can help us on her first day xD 👋

File: source/deployment/cluster.rst (hunk 1 above)
Comment (aaronrothschild) [reply source/deployment/cluster.rst]: @amyblais Interesting, do we use Canadian English or American? for future reference.

File: source/deployment/cluster.rst (hunk 2 above)
Comment (aaronrothschild) [reply source/deployment/cluster.rst]: I re-worded it to simplify.

Comment (aaronrothschild) [reply source/deployment/cluster.rst]: @justinegeffen Please, let me know if you have any suggestions.

File: source/deployment/cluster.rst [hunk 3]
```diff
@@ -293,6 +293,14 @@ Mattermost runs periodic tasks via the `job server <https://docs.mattermost.com/
 
 :ref:`Run all job servers <command-line-tools-mattermost-jobserver>` with ``--noschedule flag``, then set ``JobSettings.RunScheduler`` to ``true`` in config.json for all app servers in the cluster. The cluster leader will then be responsible for scheduling recurring jobs.
 
+Plugins and High Availability
+^^^^^^^^^^^^^^^^
+
+As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. 
+
+Note a slight behaviour change in 5.15:
+When you re-install a plugin in 5.14, it retained the last `Enabled` or `Disabled` state as it was before. As of 5.15, when you re-install a plugin it's initial state will now be DISABLED.
```
Comment (amyblais) [inline source/deployment/cluster.rst]:
```suggestion
When you re-install a plugin in 5.14, it retained the last "Enabled" or "Disabled" state as it was before. As of 5.15, a plugin's initial state will now be DISABLED when you re-install it.
```

**Comment by aaronrothschild** [comment]
@ali-farooq0 All good?  I can't add @justinegeffen as a reviewer because she is not part of the MM org it seems.

File: source/deployment/cluster.rst [hunk 4]
```diff
@@ -293,6 +293,14 @@ Mattermost runs periodic tasks via the `job server <https://docs.mattermost.com/
 
 :ref:`Run all job servers <command-line-tools-mattermost-jobserver>` with ``--noschedule flag``, then set ``JobSettings.RunScheduler`` to ``true`` in config.json for all app servers in the cluster. The cluster leader will then be responsible for scheduling recurring jobs.
 
+Plugins and High Availability
+^^^^^^^^^^^^^^^^
+
+As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. 
+
+Note a slight behaviour change in 5.15:
+When you re-install a plugin in 5.14, it retained the last "Enabled" or "Disabled" state as it was before. As of 5.15, a plugin's initial state will now be DISABLED when you re-install it.
```
Comment (amyblais) [inline source/deployment/cluster.rst]:
```suggestion
When you re-install a plugin in 5.14, it retained the last "Enabled" or "Disabled" state as it was before. As of 5.15, a plugin's initial state will now be disabled when you re-install it.
```

**Comment by justinegeffen** [comment]
Thanks for tagging me! Apologies for the delay. :) My suggestion for the wording is this:
"When you reinstall a plugin in 5.14, the previous “Enabled" or "Disabled" state is retained. As of 5.15, a reinstalled plugin's initial state is “Disabled”."

If that works for everyone, I'm happy to update it once I'm in the MM organization.

**Comment by ali-farooq0** [comment]
> Thanks for tagging me! Apologies for the delay. :) My suggestion for the wording is this:
> "When you reinstall a plugin in 5.14, the previous “Enabled" or "Disabled" state is retained. As of 5.15, a reinstalled plugin's initial state is “Disabled”."
> 
> If that works for everyone, I'm happy to update it once I'm in the MM organization.

@justinegeffen Works for me. 👌

**Pull request merged**